his/core/application-integration-3270designer-2.md

```diff
@@ -0,0 +1,44 @@
+---
+title: What is the 3270 Design Tool
+description: Learn about creating green-screen applications with the 3270 Design Tool for Azure Logic Apps.
+ms.prod: host-integration-server
+ms.topic: conceptual
+ms.date: 10/17/2023
+
+#CustomerIntent: As a programmer, I need to create metadata for the 3270 connector in Azure Logic Apps using the 3270 Design Tool.
+---
+
+# What is the 3270 Design Tool? 
+
+To create mainframe metadata for 3270 stream-based artifacts to use in Azure Logic Apps, you can use the 3270 Design Tool. This tool is a desktop application that helps you define metadata used by the IBM 3270 connector in Azure Logic Apps to navigate and interact with your 3270 applications.
+
+:::image type="content" source="media/la-3dtdesigner-main-view1.png" alt-text="HIS Designer for Azure Logic Apps":::
+
+## Why use the 3270 Design Tool to interact with 3270 applications?
+
+In the SNA days, the display management protocol used to facilitate the communication between remote end-users and a centralized mainframe was called the 3270 data stream. At the end user's location in an SNA network was a device referred to as a 3270 terminal. Because so many SNA and 3270 applications existed, customer looked at integrating their SNA protocol into their IP backbone. The technology used to move from SNA 3270 applications to TCP/IP is called TN3270, short for Telnet 3270. Extracted from [Introduction to the 3270 terminal](https://www.ibm.com/docs/en/zos-basic-skills?topic=enhanced-introduction-3270-terminal).
+
+3270 Applications are pervasive in IBM Mainframe environments. To access these applications, which oftentimes have business logic embedded with the screens, separating the screen and business logic might not be possible or maybe you no longer have information for how the applications work. The 3270 Design tool helps to parse and process these applications so the screen information can be reused.
```

Comment:
```suggestion
3270 applications are pervasive in IBM mainframe environments. To access these applications, which often have business logic embedded with the screens, separating the screen and business logic might not be possible, or maybe you no longer have information for how the applications work. The 3270 Design Tool helps parse and process these applications so that you can reuse the screen information.
```

Reply:
@haroldcampos: Please review this suggested edit? Thanks!